validation/poststart_fail.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	tap "github.com/mndrix/tap-go"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/opencontainers/runtime-tools/specerror"
+	"github.com/opencontainers/runtime-tools/validation/util"
+	uuid "github.com/satori/go.uuid"
+)
+
+func main() {
+	t := tap.New()
+	t.Header(0)
+
+	config := util.LifecycleConfig{
+		Actions: util.LifecycleActionCreate | util.LifecycleActionStart | util.LifecycleActionDelete,
+		PreCreate: func(r *util.Runtime) error {
+			r.SetID(uuid.NewV4().String())
+			g := util.GetDefaultGenerator()
+			poststart := rspec.Hook{
+				Path: fmt.Sprintf("%s/%s/bin/false", r.BundleDir, g.Spec().Root.Path),
+				Args: []string{"false"},
+			}
+			g.AddPostStartHook(poststart)
```

Comment:
I think you also want a second hook which writes to a file, and a post-`RuntimeLifecycleValidate` check to ensure that file was never written.  Similarly with `SetProcessArgs`, you'll want to set something there so we can confirm that the runtime didn't actually execute the user-specified program.

Reply:
+1 for this, I'll update these.